elixir/apps/api/lib/api/controllers/actor_controller.ex

```diff
@@ -0,0 +1,56 @@
+defmodule API.ActorController do
+  alias Domain.Actors
+  import API.ControllerHelpers
+  use API, :controller
```

Comment:
The order should be `use`, `import`, `alias`

Reply:
I always forget this order.  Thank you for the reminder.

Reply:
To remember you can think "what is more polluting for the module":
- `use` can add functions via macro, aliases, imports and pretty much anything else, so it comes first;
- `import` adds functions to module namespace and a compile-time dependency, so comes second;
- `alias` just adds a module alias, third;
- `require` adds a compile-time dependency so that macros can be used but doesn't pollute the namespace at all, comes last.